daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -615,3 +620,70 @@ func unwrapEvents(events []wrappedEvent) []*cloudwatchlogs.InputLogEvent {
 	}
 	return cwEvents
 }
+
+func newEventBatch() *eventBatch {
```

Comment:
Perhaps in a future rewrite we could pass in `maximumLogEventsPerPut` and `maximumBytesPerPut` (they feel like a property of the `eventsBatch`);

```go
func newEventBatch(maxEvents uint, maxBytes uint) *eventBatch { 
```